Makefile-rpm.mk

```diff
@@ -0,0 +1,29 @@
+NAME    := raft
+SRC_EXT := gz
+SOURCE   = v$(VERSION).tar.$(SRC_EXT)
+
+GIT_SHA1        := $(shell git rev-parse HEAD)
+GIT_SHORT       := $(shell git rev-parse --short HEAD)
+GIT_NUM_COMMITS := $(shell git rev-list HEAD --count)
+
+GIT_INFO := $(GIT_NUM_COMMITS).g$(GIT_SHORT)
+
+BUILD_DEFINES := --define "%relval $(GIT_INFO)"
+RPM_BUILD_OPTIONS := $(BUILD_DEFINES)
+
+%.$(SRC_EXT): %
+	rm -f $@
+	gzip $<
```

Comment:
This is a tempting recipe that I have started to write more than once but It's not `gzip` if `$(SRC_EXT)` isn't `gz`.

These need writing as individual recipes to create each type of `$(SRC_EXT)` and should be in `packaging/Makefile_packaging.mk`.

Reply:
Removed this as Makefile_packaging.mk already handling.  Adding .bz2 and xz compression to next update